features/networking/service.feature

```diff
@@ -479,3 +479,56 @@ Feature: Service related networking scenarios
     Then the step should succeed
     And the output should not contain "mtu-too-small"
     """
+
+  # @author zzhao@redhat.com
+  # @case_id OCP-33848
+  @admin
+  @destructive
+  Scenario: User can expand the nodePort range by patch the serviceNodePortRange in network
+    Given I have a project
+    And evaluation of `rand(32676..33000)` is stored in the :port clipboard
+    Given as admin I successfully merge patch resource "networks.config.openshift.io/cluster" with:
+      | {"spec":{"serviceNodePortRange": "30000-33000"}} |
+    When I obtain test data file "networking/nodeport_service.json"
+    And I wait up to 120 seconds for the steps to pass:
+    """    
+    When I run oc create over "nodeport_service.json" replacing paths:
+      | ["items"][1]["spec"]["ports"][0]["nodePort"] | <%= cb.port %> |
+    Then the step should succeed
+    """
+    Given the pod named "hello-pod" becomes ready
+    And evaluation of `pod('hello-pod').node_ip(user: user)` is stored in the :hostip clipboard
+
+    Given I obtain test data file "networking/list_for_pods.json"
+    When I run oc create over "list_for_pods.json" replacing paths:
+      | ["items"][0]["spec"]["replicas"] | 1 |
+    Then the step should succeed
+    And a pod becomes ready with labels:
+      | name=test-pods |
+    When I execute on the pod:
+      | curl | <%= cb.hostip %>:<%= cb.port %> |
+    Then the step should succeed
+    And the output should contain:
+      | Hello OpenShift! |
+    When I run the :delete client command with:
+      | object_type | service |
+      | object_name_or_id | hello-pod |
+    Then I wait for the resource "service" named "hello-pod" to disappear
```

Comment:
Above 4 lines can be shorten as `Given I ensure "xxx" service is deleted`

Reply:
updated. thanks